### PR TITLE
Enyo 3309 yeramchoi

### PR DESCRIPTION
**Pull request by Djspaceg**

### CODE REVIEW

File: src/Notification/Notification.less [hunk 1]
```diff
@@ -22,10 +27,10 @@
 		&::before {
 			content: '';
 			display: block;
-			background-color: @moon-neutral-bg-color;
-			border: 6px solid @moon-black;
-			border-radius: 30px;
-			opacity: 0.85;
```
Comment (Djspaceg) [inline src/Notification/Notification.less]: @yeram What was the reasoning for removing this element's opacity, only to replace it with the background color and border color alpha? Borders stack on top of background. Ex: a 50% alpha border on top of a 50% alpha background results in an effective 75% total opacity for the border.

File: src/Notification/Notification.less [hunk 2]
```diff
@@ -1,4 +1,11 @@
 /* Popup.css */
+.moon {
+	.moon-notification {
+		padding: 27px 39px 27px 39px;
+		display: table;
+	}
+}
+
```
Comment (Djspaceg) [inline src/Notification/Notification.less]: I assume this is just for specificity precedence?

File: src/Notification/Notification.less [hunk 3]
```diff
@@ -6,11 +13,9 @@
 	position: absolute;
 	left: 50%;
 	bottom: 0;
-	max-width: 70%;
-	padding: @moon-popup-padding;
+	max-width: 1083px;
```
Comment (Djspaceg) [inline src/Notification/Notification.less]: Lets make this 1080px

File: src/Notification/Notification.less (hunk 2 above)
Comment (yeram) [inline src/Notification/Notification.less]: Hmm, I saw padding in .moon-notification is overridden by padding in .moon's.

http://10.186.120.46/~yeram.choi/dist/

File: src/Notification/Notification.less (hunk 3 above)
Comment (yeram) [inline src/Notification/Notification.less]: got it!

File: src/Notification/Notification.less (hunk 1 above)
Comment (yeram) [inline src/Notification/Notification.less]: Ah, I couldn't consider side effect of color alpha. I just implemented based on my experience about performance between opacity and color alpha.
I couldn't remember exact number of performance difference and don't know it is still presented, so it's better to keep your implementation for preventing side effect of color alpha.

**Comment by lishammel** [comment]
Closing PR as jira has been closed.

**Pull request closed without merging**